improve the wording of theming docs (#6767)
plone/volto: docs/source/theming/using-third-party-themes.md

## Setting up the theme

Customizing the base theme is a special use case in Volto.
The original file is in Volto at {file}`volto/src/theme.js`.
This is the file to be customized.
In the {file}`customizations` folder, override it as {file}`customizations/@root/theme.js`, using the `@root` alias to avoid writing the full path.

> Add the following code to this file.

Reviewer: ```suggestion
Edit the imports in this file to align with the following code.
```

## Setting up the theme

> In your Volto project, customize the file `src/theme.js`, creating a `customization/@root/theme.js` file:

Reviewer: This sentence doesn't make sense to me. How do I customize one file by creating another file?

Reviewer: This is closer, however the subsequent diff code-block doesn't make sense for what should go into `customizations/@root/theme.js`.

```suggestion
Customize Volto's {file}`src/theme.js` by creating a file in the {file}`customizations` folder to override the original.
Create the file {file}`customizations/@root/theme.js`.
```

Author: The contents here and in the `add-ons/theme.md` file should be the same, as the procedure is the same. So if you think that the first change is clear enough, we can copy it here.

Reviewer: PR #6784 merged!

+ import '../theme/site.scss';
```

> Then, copy the `theme.config` file present in core Volto (`core/packages/volto/theme/theme.config`) to your add-on's `src/theme/` folder and change the following and the needed variables:

Reviewer: ```suggestion
Then, copy the {file}`theme.config` file from core Volto ({file}`core/packages/volto/theme/theme.config`) to your add-on's {file}`src/theme/` folder.
Change the following variable as shown.
```

### Use Sass loader

> ```{note}
> This is not required anymore since Volto 18.

Reviewer: ````suggestion
```{versionremoved} Volto 18
This section is no longer required since Volto 18.
````
